flight/Libraries/math/misc_math.h

```diff
@@ -78,6 +78,100 @@ static inline bool IS_NOT_FINITE(float x) {
 	return (!isfinite(x));
 }
 
+/* Following functions from fastapprox https://code.google.com/p/fastapprox/
```

Comment:
@hernick-qc Good to know about the license. I'm not sure that we can arrive at the conclusion yet, since the second point is that an argument could be made that this isn't really miscellaneous math so much as it is optimization math. This code doesn't introduce new functionality, it improves on existing functions.

I like the notion of having an optimization library, but it's up to the author.